src/components/StatusIcon/__tests__/StatusIcon.spec.js

```diff
@@ -3,35 +3,70 @@
  * @copyright IBM Security 2019
  */
 
-import { shallow } from 'enzyme';
+import { render } from '@testing-library/react';
 import React from 'react';
 
-import StatusIcon, { STATUS, SIZE } from '../StatusIcon';
+import StatusIcon, { namespace, STATUS, SIZE } from '../StatusIcon';
 
-describe('StatusIcon', () => {
-  let statusIcon;
+import { carbonPrefix } from '../../../globals/namespace';
 
-  beforeEach(() => {
-    statusIcon = shallow(<StatusIcon />);
-  });
+describe('StatusIcon', () => {
+  // Should check for `undefined` status as well:
+  STATUS.push(undefined);
```

Comment:
Instead of pushing to the status, which could permanently change the array for other tests, I would create a new array that combines the `STATUS` array and `undefined`.

Reply:
And actually, now that I'm seeing the rest of the tests. I think it's easier to just have special single tests for the undefined status.

Reply:
Thanks @diego-codes that's a good point! I just updated the tests to avoid pushing the array & have separate tests for `status={undefined}` 👍